charts/tabulation/templates/deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-tabulation-app
+  labels:
+    app: tabulation-app
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: tabulation-app
+  template:
+    metadata:
+      annotations:
+      labels:
+        app: tabulation-app
+    spec:
+      imagePullSecrets:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: ghcr.io/cdcgov/phdi/tabulation:{{ .Values.image.tag }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: {{ .Chart.Name | trunc 15 }}
```

Comment:
Is the tabulation name too long for K8s? 😆

Reply:
its a 15 character limit apparently... lol